templates/lookup/ifupdown__dynamic_packages.j2

```diff
@@ -0,0 +1,21 @@
+{% set ifupdown__tpl_packages = [] %}
+{% if ifupdown__combined_interfaces|d() %}
+{%   for interface, params in ifupdown__combined_interfaces.iteritems() %}
+{%     if (params.state|d('present') == 'present' and params.type|d()) %}
+{%       if params.type == 'bridge' %}
+{%         set _ = ifupdown__tpl_packages.append('bridge-utils') %}
+{%       elif params.type == 'vlan' %}
+{%         set _ = ifupdown__tpl_packages.append('vlan') %}
+{%       elif params.type == 'bond' %}
+{%         if ansible_distribution_release == 'wheezy' %}
+{%           set _ = ifupdown__tpl_packages.append('ifenslave-2.6') %}
+{%         else %}
+{%           set _ = ifupdown__tpl_packages.append('ifenslave') %}
+{%         endif %}
+{%       endif %}
+{%     endif %}
+{%   endfor %}
+{% endif %}
+{% if ifupdown__tpl_packages %}
```

Comment:
@drybjed I am not able to reproduce this with 2.0, 2.1.3 nor 2.2. Can you retest? I made the change locally and removed the condition and will continue to test it and open a PR later.